### DISCUSSION TITLE
Why `lightBlue`?

### DISCUSSION THREAD

**Discussion by ianstormtaylor** [tailwindlabs/tailwindcss#4077] [Ideas]
Just wondering… the color names are all so great and terse. I can never remember that `lightBlue` is even an option, and when I do I remember that the camel case is awkward. 

Why not something that matches the rest like `sky`?

**Comment by bagsa** [comment]
Ii

**Comment by adamwathan** [comment]
I was thinking about this myself again last night, hah. Sky is nice! At the time I think we just had too many things going on and settled for it in order to focus brain power on other stuff. Since the general recommendation is to alias your colors (I'd pull in light blue as just "blue") it didn't feel super important. There was precedent for two word names with coolGray for example too so whatever.

I think we can alias to sky in a future update though and stop documenting it as lightBlue, much more elegant *chef's kiss*

**Reply by ianstormtaylor** [reply]
Super fair on prioritization. And if it can end up being aliased in the future that would be awesome!
